src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -60,7 +60,8 @@ def compress(self):
         """
         modules_to_compress = self.detect_modules_to_compress()
         for layer, config in modules_to_compress:
-            self._instrument_layer(layer, config)
+            module = self._instrument_layer(layer, config, self.bound_model)
+            layer.module = module
```

Comment:
do not suggest to update `layer.module` with wrapper module. this would lead to inconsistent result of the call `detect_modules_to_compress`

Reply:
Removed.